baxter_collaboration_lib/test/README.md

```diff
@@ -0,0 +1,30 @@
+# Testing
+
+### Using [Catkin Tools](https://catkin-tools.readthedocs.io/en/latest/)
+
+1. Navigate to your workspace: `cd ros_ws`
+2. `REPO_NAME=baxter_collaboration`
+3. Compile and test: `catkin build "$REPO_NAME"_lib && catkin build "$REPO_NAME"_lib --catkin-make-args run_tests`
+4. Check the results of above: `catkin_test_results build/"$REPO_NAME"_lib || (cat build/"$REPO_NAME"_lib/test_results/"$REPO_NAME"_lib/*.xml ; false)`
+
+Troubleshooting:
+* If build fails, make sure prerequisites are installed: [`aruco_ros`](https://github.com/ScazLab/aruco_ros), `nlopt`, [`trac_ik`](https://bitbucket.org/alecive/trac_ik), [`baxter_common`](http://sdk.rethinkrobotics.com/wiki/Workstation_Setup#adeedda5360641914fe9c5d681c30026) and [`moveit`](http://moveit.ros.org/install/).
```

Comment:
When Andrew and I were trying to get him set up, he was unable to build baxter_collaboration without moveit. On the other hand, his trac_ik package was not building, so perhaps if his trac_ik builds, moveit may not be required. So I can defer this till tomorrow?